client/boot/app.js

```diff
@@ -0,0 +1,56 @@
+// Initialize localStorage polyfill before any dependencies are loaded
+require( 'lib/local-storage' )();
+if ( process.env.NODE_ENV === 'development' ) {
+	require( 'lib/wrap-es6-functions' )();
+}
+
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import { invoke } from 'lodash';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import {
+	configureReduxStore,
+	locales,
+	setupMiddlewares,
+	utils
+} from './common';
+import createReduxStoreFromPersistedInitialState from 'state/initial-state';
+import detectHistoryNavigation from 'lib/detect-history-navigation';
+import userFactory from 'lib/user';
+
+const debug = debugFactory( 'calypso' );
+
+const boot = currentUser => {
+	debug( "Starting Calypso. Let's do this." );
+
+	const project = require( `./project/${ config( 'project' ) }` );
```

Comment:
Looks great :) I'm glad to see such a simple solution!
Do you think we'll see issues outside of Webpack bundles? Shall we cross that bridge when we get to it or do you think it's worth testing out now?